semantic-source/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.0.0.2
+
+- Adds an `NFData` instance for `Source`.
```

Comment:
I think this means it should be `0.0.1.0`.

Reply:
Good catch.